### PR TITLE
HW5 if-else-switch task

### PR DESCRIPTION
**Pull request by andreamamchur**

### CODE REVIEW

File: Exercises/if-else-switch/if-else-switch.js [hunk 1]
```diff
@@ -4,11 +4,47 @@ const codeToMissingComplexityElement = {
   2: 'special character',
   3: 'cannot be empty'
 };
+let specialCharacterCheck = false;
+const specialCharacters = ['!', '_', '-', '%', '(', ')', '#', '*', '{', '}'];
+let arrayResult = [];
 
 function getMissingComplexityElementCode(password) {
+  for (const char of password) {
```
Comment (lizarusi) [inline Exercises/if-else-switch/if-else-switch.js]: better to iterate through `specialCharacters` array

File: Exercises/if-else-switch/if-else-switch.js [hunk 2]
```diff
@@ -4,11 +4,47 @@ const codeToMissingComplexityElement = {
...
+    if (specialCharacters.includes(char)) {
+      specialCharacterCheck = true;
+    }
+  }
 
+  if (password.length < 8) {
+    arrayResult.push(0);
+  }
+  if (password === password.toLowerCase()) {
```
Comment (lizarusi) [inline Exercises/if-else-switch/if-else-switch.js:21]: 👍

File: Exercises/if-else-switch/if-else-switch.js [hunk 3]
```diff
@@ -4,11 +4,47 @@ const codeToMissingComplexityElement = {
...
+    arrayResult.push(3);
+  }
 }
 
-function getErrorMessage() {
-  return false;
+function getErrorMessage(array) {
+  for (const char of array) {
+    // eslint-disable-next-line default-case
+    switch (char) {
```
Comment (lizarusi) [inline Exercises/if-else-switch/if-else-switch.js]: the idea here, is to get value of key i.e. `length`, `empty`... and use it in switch cases.

**Review by lizarusi — CHANGES_REQUESTED** [review]
awesome work!

File: Exercises/if-else-switch/if-else-switch.js [hunk 4]
```diff
@@ -4,11 +4,47 @@ const codeToMissingComplexityElement = {
...
+let specialCharacterCheck = false;
+const specialCharacters = ['!', '_', '-', '%', '(', ')', '#', '*', '{', '}'];
+let arrayResult = [];
 
 function getMissingComplexityElementCode(password) {
+  for (const char of specialCharacters) {
+    if (password.includes(char)) {
+      specialCharacterCheck = true;
+    }
+  }
```
Comment (Mike-Savin) [inline Exercises/if-else-switch/if-else-switch.js:16]: Looks good. You can also try to use `array.some` method here if you want:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some

**Review by Mike-Savin — APPROVED** [review]
👍

**Pull request merged**